other/parser_build.yml

```diff
@@ -0,0 +1,35 @@
+trigger: none
```

Comment:
You should add here an expression path to be triggered automatically when files in the cmake-configurator are modified.
Also, how did you test this YAML file? Did you create a dedicated pipeline in your Azure account?